pep-0710.rst

```diff
@@ -140,6 +140,11 @@ Additionally, the user:password section of the URL MAY be a well-known,
 non-security sensitive string. A typical example is ``git`` in the case of an
 URL such as ``ssh://git@gitlab.com``.
 
+The value of the ``index_url`` key MUST be a base URL of a Python package index
+used to download the given distribution package, such as
+``https://pypi.org/simple``. The recorded URL SHOULD point to a repository
```

Comment:
Would ``https://pypi.org/simple`` have a trailing ``/``?

Reply:
Added a trailing slash, also to examples. Should we be more explicit about the trailing slash and its use here?